src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -274,6 +227,35 @@ public void analyzeUserUsage() throws IOException {
         log.info("User usage analysis completed!");
     }
 
+    /**
+     * {@code GET /wrap-token-stats}: Wrap token stats
+     *
+     * @throws IOException
+     */
+    @DeleteMapping(path = "/wrap-token-stats")
+    public void wrapTokenStats() throws IOException {
+        Instant current = Instant.now();
+        // Update tokenStats in database
+        updateTokenStats(current);
```

Comment:
Would be interesting to know whether this is faster than calculating based on the text file you generated bellow.